packages/expo-file-system/android/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,21 @@
 
-<manifest package="expo.modules.filesystem">
-
-</manifest>
-  
\ No newline at end of file
+<manifest package="expo.modules.filesystem"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.STORAGE" />
+    <application>
+        <provider
+            android:name="android.support.v4.content.FileProvider"
+            android:authorities="${applicationId}.provider"
```

Comment:
if we're defining this name, can we name it something more specific to this expo module, like `FileSystemProvider`? would be good to try and avoid clashes with other providers people might have defined in their apps.